manifests/quick-start/base/executor/kubelet/executor-role.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: executor
+  annotations:
+    workflows.argoproj.io/description: |
+      Recomended minimum permissions for `kubelet` executor.
```

```suggestion
      Recommended minimum permissions for `kubelet` executor.
```

Same in other places